### ISSUE TITLE
Remove `installinstallmacos` after/before script runs

### ISSUE THREAD

**Issue by wegotoeleven** [grahampugh/erase-install#61]
Hey. I've experienced an issue involving a machine that connected to a VPN that restricted access to Github. This restriction essentially prohibited the `erase-install` script from downloading the `installinstallmacos.py` script, and in it's place, was a blank file. From that point onward, every time the `erase-install` script was run, even when not connected to the restrictive VPN, the blank file persisted, and the script failed. I needed to manually delete the `installinstallmacos.py` script, at which point the `erase-install` script correctly downloaded the `installinstallmacos.py` script, and everything worked as it should.

Tl;dr; can we delete the `installinstallmacos` script before/after each run?

**Comment by grahampugh** [comment]
There's two existing solutions for that already:

- use the package rather than manually using the script, which includes installinstallmacos.py and would overwrite any broken version 

- use the --force-curl flag which will download installinstallmacos.py fresh every time. 

Cheers
Graham

**Comment by wegotoeleven** [comment]
Nice! --force-curl sounds awesome. Thanks!

**Comment by wegotoeleven** [comment]
I've made a [PR](https://github.com/grahampugh/erase-install/pull/62) to add this information into the README because why not. If you're happy with this, then feel free to merge :)
